substrate/bin/utils/chain-spec-builder/tests/test.rs

```diff
@@ -194,154 +292,83 @@ fn test_add_code_substitute() {
 	assert_output_eq_expected(true, SUFFIX, "tests/expected/add_code_substitute.json");
 }
 
-#[docify::export]
 #[test]
-fn create_default() {
-	let runtime_path = substrate_test_runtime::WASM_BINARY_PATH.expect("a valid path. qed.");
-	let output_path = "/tmp/test.create_default.json";
-	let expected_output = "tests/expected/doc/create_default.json";
-	let bin_path = env!("CARGO_BIN_EXE_chain-spec-builder");
-	run_cmd!(
-		$bin_path -c $output_path create -r $runtime_path default;
-		sed -i "/\"code\":/d" $output_path;
-		diff $output_path $expected_output;
-		rm $output_path;
-	)
-	.expect("cmd success. qed.");
+#[docify::export]
```

Comment:
This would only export content of the function, will look better in doc.
```suggestion
#[docify::export_content]
```

Reply:
As mentioned [here](https://github.com/paritytech/polkadot-sdk/pull/5954#discussion_r1823353113), this will result in failing docs test. Will resolve this since the same subject was discussed in the linked comment, which is still open and can continue the discussion towards better looking docs.

Reply:
I missed that comment, sorry. Answered there.

Reply:
Setting `doctest = true` in `Cargo.toml` made this possible, so I simplified the tests even further: [995af51](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/995af51fda7dcd9e1242b48df211e7bd06e1d0bd). I think this was what we aimed for all along. The tests weren't successful either because of a macro logic error before this commit. I am quite pleased with this right now, thanks for the suggestions and help!

Reply:
Yeah, this looks really good now. _Pure_ bash look and tested. Thanks for exploring this.